README.md

```diff
@@ -358,11 +358,11 @@ at runtime.
 ```
 Properties                  JSObject                    HiddenClass (Map)
 +-----------+              +-----------------+         +----------------+
-|property1  |<------+      | HiddenClass     |-------->| bit field1     |
+|property1  |<------+      | HiddenClass     |-------->| bit field1      |
```

Comment:
Thanks for the PR! Just wondering about this change and the others below it. It does not look right to me when I view it in a markdown viewer, is there something that I'm missing?

Reply:
I'm not sure what you see, but It looked like the left to me, and I changed it to the right. 😄 
![change](https://user-images.githubusercontent.com/16755893/34367374-f3fd00e2-eacf-11e7-90e2-835589ac6271.png)